app/assets/stylesheets/layout/_base.sass

```diff
@@ -31,7 +31,7 @@
   @include default-transition
   display: grid
   grid-template-rows: auto 1fr
-
+  height: 100%
```

Comment:
These changes causes some weird automated scrolling effects in Safari. When scrolling down, the page scrolls up automatically afterwards. Could you please check this?
Also, why is this always applied and not only for mobile?